tuftool/src/error.rs

```diff
@@ -221,6 +221,16 @@ pub(crate) enum Error {
         backtrace: Backtrace,
     },
 
+    #[snafu(display(
+        "Root was signed with less than {} signatures: {}",
```

Comment:
Nit: It may be slightly clearer to say something like:

`Root was signed with {} signatures; it must be signed with at least {}` or something similar